usaspending_api/search/v2/urls.py

```diff
@@ -14,7 +14,7 @@
     url(r"^spending_by_award_count", SpendingByAwardCountVisualizationViewSet.as_view()),
     url(r"^spending_by_award", SpendingByAwardVisualizationViewSet.as_view()),
     url(r"^spending_by_category/", include("usaspending_api.search.v2.urls_spending_by_category")),
-    url(r"^spending_by_category", SpendingByCategoryVisualizationViewSet.as_view()),
+    url(r"^spending_by_category$", SpendingByCategoryVisualizationViewSet.as_view()),
```

Comment:
Cause of your tests failing. Don't have an issue adding this, but would need to verify that this won't mess up frontend like it did the test case.

Reply:
Yeah, suspected this was going to cause an issue but having a local issue not being able to reliably execute the integration tests at the moment so hard for me to quickly diagnose and remedy. Working through now.

I like adding the `$` so it disallows a caller to hit `.../spending_by_category/anytexthere` and still weirdly being able to hit `.../spending_by_category`. Playing with the tests now to see if we can have best of both worlds.

Reply:
Actually, when I take off the `$` neither the `.../recipient_duns` nor new `.../recipient` endpoints get directly hit which was the original reason I went with the `$`.

Reply:
Looks like frontend code only uses the spending_by_category/{category_type} version of this endpoint so should be good